src/components/Map.tsx

```diff
@@ -242,82 +290,90 @@ const MapContainer = (props) => {
     }
   };
 
-  const parseLngLat = (s) => {
+  const parseLngLat = (s: string) => {
     const [lat, lng] = s.split(",").map((n) => Number(n));
     return { lng, lat };
   };
 
-  const lngLatToString = (lngLat) =>
+  const lngLatToString = (lngLat: { lat: number; lng: number }) =>
```

Comment:
It's the Coords type, isn't it?

Reply:
You're right! We actually don't need our own `type Coords` at all, we could just import a lat/lng type from `react-map-gl`, or if it's not there we should get it from `maplibre-gl`. Haven't checked what the name is, but there has got to be one.